lib/src/main/java/uk/gov/di/ipv/core/library/config/EnvironmentVariable.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.di.ipv.core.library.config;
+
+public enum EnvironmentVariable {
+    ACCESS_TOKENS_TABLE_NAME,
```

Comment:
Not something for this PR, but another nice improvement we could look into is seeing which of these env vars are actually needed. I have a feeling we have a few where maybe its not necessary and is just a bit too "cover-configured", or could be moved over to SSM perhaps.

Reply:
Yeah, I think this work has shown how many Environment Variables we have, and can then plan to remove those that are not needed.